### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by morazow**

### CODE REVIEW

File: doc/changes/changes_1.0.1.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Azure Data Factory Functions 1.0.1, released 2021-09-21
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_1.0.1.md:1]: Should we update the date?

**Review by AnastasiiaSergienko — COMMENTED** [review]
One tiny comment + maybe we should fix the linter's finding?

**Comment by morazow** [comment]
For that, I am waiting @pj-spoelders. I was not sure if Exception could wrap another exception. From docs, I could not find it, they all seemed to only accept exception message or I am looking to wrong docs.

**Comment by morazow** [comment]
Created #12 for it.

**Pull request merged**